MAUI/DataForm/Data-annotations.md

```diff
@@ -0,0 +1,374 @@
+---
+layout: post
+title: Data annotations in .NET MAUI DataForm control | Syncfusion
+description: Learn about the Data annotations support in Syncfusion Maui DataForm (SfDataForm) control in mobile and desktop applications from a single shared codebase.
+platform: maui
+control: SfDataForm
+documentation: UG
+---
+
+# Data annotations in .NET MAUI DataForm (SfDataForm)
+
+The data form supports the following attribute to handle the data, and these attributes can be accessible using `System.ComponentModel.DataAnnotation` assembly.
+
+## Display attribute
+
+<table>
+<tr>
+<th>Property</th>
+<th>Details</th>
+</tr>
+<tr>
+<td>
+{{'[Name](https://learn.microsoft.com/en-us/dotnet/api/system.componentmodel.dataannotations.displayattribute.name?view=net-7.0)' |  markdownify }}
+</td>
+<td>
+Specifies the label text.
+</td>
+</tr>
+<tr>
+<td>
+{{'[GroupName](https://learn.microsoft.com/en-us/dotnet/api/system.componentmodel.dataannotations.displayattribute.groupname?view=net-7.0)' |  markdownify }}
+</td>
+<td>
+Specifies the group name which groups the fields in the data form.
+</td>
+</tr>
+<tr>
+<td>
+{{'[ShortName](https://learn.microsoft.com/en-us/dotnet/api/system.componentmodel.dataannotations.displayattribute.shortname?view=net-7.0)' |  markdownify }}
+</td>
+<td>
+Specifies the label text. It takes higher priority than Name.
+</td>
+</tr>
+<tr>
+<td>
+{{'[AutoGenerateField](https://learn.microsoft.com/en-us/dotnet/api/system.componentmodel.dataannotations.displayattribute.autogeneratefield?view=net-7.0)' |  markdownify }}
+</td>
+<td>
+Specifies whether the field should be auto generated or not. 
+</td>
+</tr>
+<tr>
+<td>
+{{'[ResourceType](https://learn.microsoft.com/en-us/dotnet/api/system.componentmodel.dataannotations.displayattribute.resourcetype?view=net-7.0)' |  markdownify }}
+</td>
+<td>
+Specifies the Resources File (.Resx) which is used to localize the Display attribute of `Name`, `ShortName`, `GroupName`, `Prompt` and `ItemsSource` values.
```

Comment:
Specifies the Resources File (.Resx), which is used to localize the Display attribute of the `Name,` `ShortName,` `GroupName,` `Prompt,` and `ItemsSource` values.

Reply:
, added